pages/reference/hardware/wifi-dongles.md

```diff
@@ -8,6 +8,14 @@ excerpt: WiFi adapters known to work with {{ $names.company.lower }} devices
 {{> "meta-balena/supported-wifi-adapters" }}
 
 
+Important: Our testing process is only doing a sanity check on the features that are commonly required by balenaOS using the driver provided by the manufacturer. These tests are restricted to:
+
+- Wireless station mode can connect to an AP
+- Bluetooth can scan
+- A modem can register with a network
+
```

Comment:
I would remove the examples sanity tests as these can change with time. 

Leaving the statement as:
```
Balena's software validation is only doing a sanity check on the features that are commonly required by balenaOS using the driver provided by the manufacturer and the NetworkManager/ModemManager support. Balena cannot guarantee that a modem or chipset is reliable and ready for production. Users should run the necessary tests to completely validate their use case and work with the vendor to resolve problems. 
```

Reply:
Resolved